docs/resources/convert_v1_to_v2_resource_groups.md

```diff
@@ -0,0 +1,251 @@
+---
+subcategory: "Resource Groups"
+layout: "lacework"
+page_title: "Lacework: convert_to_newer_resource_group"
+description: |-
+  Converts old to new resource groups
+---
+
+# convert_to_newer_resource_group
+
+The new version of Resource Groups changes the Terraform syntax used to create resource groups. 
+The old version of Resource Groups defined specific filter fields for each of the Resource Group 
+types. The new version defines the `group` argument as an expression tree representing the 
+relationships between resources.
```

Comment:
Use "Resource groups provide a way to categorize Lacework-identifiable assets, including the ability to populate resource groups based on conditional statements."

from https://docs.lacework.net/api/v2/docs/#tag/ResourceGroups

Then talk about how previously resource groups only supported account based filters.